webapps/api/backend/views/vdisks.py

```diff
@@ -159,6 +159,28 @@ def set_as_template(self, vdisk):
     @log()
     @required_roles(['read', 'write'])
     @return_task()
+    @load(StorageRouter)
```

Comment:
You're loading a StorageRouter here. That doesn't make sense, since you're at the ```create``` method of the vDisk API. This should just be ```@load()``` without any argument. I would even expect it to fail.